esp/services/ws_workunits/CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ include_directories (
          ./../../smc/SMCLib
          ./../../bindings/SOAP/xpp
          ${HPCC_SOURCE_DIR}/dali/dfu
+         ${HPCC_SOURCE_DIR}/dali/ft
```

Comment:
@wangkx - actually I see this is being included, so that queryDistributedFileSystem.getSize(), can be used,
but actually it is more suitable to use IDistributedFile::getDiskSize

Reply:
@jakesmith are you talking about this one:
virtual __int64 getDiskSize(bool allowphysical,bool forcephysical)=0;       // gets the part size on disk (NB this will be the compressed size) ? compressed size? allowphysical? forcephysical?

Reply:
@wangkx - yes, what we're interested in is the disk size, as that is what is going to be downloaded. If the file is compressed - the compressed size is the disk size. We're not interested so much in what it might be if it were to be decompressed (there's a separate discussion point, as to whether downloading a compressed file has any meaning/use).

I'd pass in (true, false), i.e. allow physical check if meta size is missing, but don't force it.

Reply:
I got it.